.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["es2015", "react"],
+  "presets": [ "es2015", "react"],
```

Comment:
Not sure if this is a linting thing; I'd either add a space before the closing bracket to match, or take out the space after the opening bracket